cloud/azure/templates/azure_saml_ses/variables.tf

```diff
@@ -60,6 +60,11 @@ variable "aws_region" {
   default     = "us-east-1"
 }
 
+variable "email_provider" {
+  type        = string
+  description = "The provider to use for sending emails"
+}
```

Comment:
question (non-blocking): why do you need a default for aws but not azure?

Reply:
Good question. I think for azure we'll eventually use graph-api as the default, but since that isn't supported I left it empty for now. I'll add it to the description.